src/pages/blog/blogDetails/helperFunction.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Parses html string and finds unclosed empty or void elements, and closes them.
+ * @param htmlString
+ * @returns {string}
+ */
+
+export const searchForEmptyElement = (htmlString) => {
+    const emptyElements = [
+        "<area", "<base", "<br", "<col", "<embed", "<hr", "<img", "<input", "<keygen", "<link", "<meta", "<param", "<source", "<track", "<wbr"
+    ]
+    let str = htmlString  // string to evaluate
+    let updatedStr = ""    // string evaluated
+    let strEnd = ""   //seperates end string nnot evaluated yet
+    let finalString = ""
+
+    // function to determine if element is closed
+    function isTagClosed(elementStr) {
+        //find the corresponding closing tag after the empty element
+        let [start, ...end] = elementStr.split('>');
+         //separates rest of string, and sets that as next string to evaluate
+        strEnd = end.join(">");
+
+        // if closing tag is not closed, add "/"
+        if (start[start.length - 1] !== "/") {
+            return `${start}/>`
+        // else return original string
+        } else {
+            return `${start}>`
+        }
+    }
```

Comment:
This does not need to be it's own function. It is only called once. Refactoring it this way means that whoever is reading the below function needs to scroll up to see what this portion is doing. This would be fine if it is being used in multiple places, but in this case it is just more work. It also technically adds a few lines of code to do it this way. Just implement this logic in the below for loop.